core/domain/suggestion_services.py

```diff
@@ -165,6 +166,18 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def edit_suggestion(suggestion, change_object):
+    """Edits the submitted suggestion.
+
+    Args:
+        suggestion: obj. The suggestion to be changed.
+        change_object: dict(str, str). Object of change in suugestion.
```

Comment:
If it is an object, its type cannot be dict(str, str). You might have to recheck the type.

Reply:
ohh sorry, for that.

Reply:
done